lib/jets.rb

```diff
@@ -25,4 +27,9 @@ class Error < StandardError; end
   extend Core # root, logger, etc
 end
 
+ActiveSupport.on_load(:i18n) do
+  I18n.load_path << File.expand_path("jets/locale/en.yml", __dir__)
```

Comment:
Thinking may locate this to another file maybe as in internal Turbine and require it. Near the boot logic.

Reply:
gotcha that makes sense, will knock that out.